src/app/features/date-range/components/date-range/date-range.component.ts

```diff
@@ -0,0 +1,108 @@
+import { Component, ChangeDetectionStrategy, Input, Inject } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+import { DateRange } from '@angular/material/datepicker';
+import { MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { DateRangeData, UnavailableDates } from '../../models';
+import { DateRangeController } from '../../services';
+
+@Component({
+  selector: 'lc-date-range',
+  templateUrl: './date-range.component.html',
+  styleUrls: ['./date-range.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class DateRangeComponent {
+  @Input() selectedRangeValue!: DateRange<Date> | null;
+
+  readonly maxRentalPeriod = 10;
+  readonly day = new Date().getDate();
```

Comment:
wrong name. in js day meand monday, tuesday and so on. it should be day.
I think it would be better to have toDay date object and then use it to retriev current date information.

Reply:
done